*** /home/fujics/hoshi/elses-work/elses-v0.01-for-iguchi/xmlf90-1.2g-elses/strings/m_strings.f90	Sun Jan  4 02:31:32 2009
--- xmlf90-1.2g-elses/strings/m_strings.f90	Sat Jun  6 12:39:16 2009
***************
*** 7,13 ****
  
        module m_strings
  
!       private
  
        type string
            private
--- 7,13 ----
  
        module m_strings
  
!       public
  
        type string
            private
***************
*** 368,407 ****
  !!$          module procedure lrank_c, lrank_s
  !!$      end interface
  
- 
- 
- !---- Publically accessible entities
-       public :: string
-       public :: assignment(=),unstring
- ! Mizuho-IR
- !!$      public :: insert,replace,remove,extract,substring
- !!$      public :: repeat,index,scan,verify
- !!$      public :: operator(//)
-       public :: operator(==) ! Mizuho-IR
- ! Mizuho-IR
- !!$      public :: operator(==),operator(/=) ! Mizuho-IR
- !!$      public :: operator(<),operator(<=)
- !!$      public :: operator(>),operator(>=)
- !!$      public :: llt,lle,lge,lgt
- !!$      public :: char,len,len_trim,trim,iachar,ichar,adjustl,adjustr
-       public :: char,len_trim,trim ! Mizuho-IR
- ! Mizuho-IR
- !!$      public :: lowercase,to_lowercase,uppercase,to_uppercase
- !!$      public :: strip,len_strip
- !!$      public :: sort,rank,lsort,lrank
- !!$
- !!$      public :: resize_string,string_size,swap_strings
-       public :: string_size
- !!$      public :: trim_string,strip_string
- !!$      public :: adjustl_string,adjustr_string
- !!$      public :: insert_in_string,remove_from_string
- !!$      public :: prepend_to_string,append_to_string
-       public :: append_to_string
- !!$      public :: replace_in_string
- 
- 
- 
- 
        contains
  
  !*******************************************************************************
--- 368,373 ----
